static/js/front.js

```diff
@@ -19,12 +19,12 @@ $(function () {
   animations()
   counters()
   demo()
-  contactForm()
+  contactFormAjax()
 })
 
 // Ajax contact
-function contactForm () {
-  var form = $('.contact-form')
+function contactFormAjax () {
+  var form = $('.contact-form-ajax')

```

Comment:
just a small thing, can you return if form is null?

Reply:
Done. Please review 9ae47ed03515d8d1359dcf84d6b32da98e261ce5.